pkg/gce-pd-csi-driver/controller.go

```diff
@@ -218,7 +218,7 @@ func (gceCS *GCEControllerServer) ControllerPublishVolume(ctx context.Context, r
 		return nil, status.Error(codes.InvalidArgument, "ControllerPublishVolume Volume capability must be provided")
 	}
 
-	volKey, err := common.VolumeIDToKey(volumeID)
+	volKey, _, err := common.VolumeIDToKey(volumeID)
```

Comment:
Thinking about the case where you have pv1 and pv2 on the same disk, but different partitions.  All the consumers of those pvs must be scheduled on the same node, either by using Pod affinity on hostname, or only use a single Pod.